file_organizer_v2/src/file_organizer/cli/dedupe.py

```
@@ -0,0 +1,610 @@
+        # Scan directory
+        scan_options = ScanOptions(
+            algorithm=config.algorithm,
+            recursive=config.recursive,
+            min_file_size=config.min_size,
+            max_file_size=config.max_size,
+            file_patterns=config.include_patterns if config.include_patterns else None,
+            exclude_patterns=config.exclude_patterns if config.exclude_patterns else None,
+            progress_callback=progress_callback if has_tqdm else None,
+        )
+
+        index = detector.scan_directory(config.directory, scan_options)
```

Ruff (0.14.13)

Local variable `index` is assigned to but never used

Remove assignment to unused variable `index`

(F841)